### ISSUE TITLE
AttributeError: 'TaskLogger' object has no attribute 'log_debug' 

### ISSUE THREAD

**Issue by danli349** [KrishnaswamyLab/PHATE#138]
Hello:
When I run the example codes:
```
import phate
tree_data, tree_clusters = phate.tree.gen_dla()
phate_operator = phate.PHATE(k=15, t=100)
tree_phate = phate_operator.fit_transform(tree_data)
```
There is an `AttributeError`:
```
Calculating PHATE...
  Running PHATE on 3000 observations and 200 variables.
  Calculating graph and diffusion operator...
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Cell In[10], line 2
      1 phate_operator = phate.PHATE(n_jobs=-2)
----> 2 tree_phate = phate_operator.fit_transform(tree)

File ~/.local/lib/python3.8/site-packages/phate/phate.py:947, in PHATE.fit_transform(self, X, **kwargs)
    926 """Computes the diffusion operator and the position of the cells in the
    927 embedding space
    928 
   (...)
    944     The cells embedded in a lower dimensional space using PHATE
    945 """
    946 with _logger.task("PHATE"):
--> 947     self.fit(X)
    948     embedding = self.transform(**kwargs)
    949 return embedding

File ~/.local/lib/python3.8/site-packages/phate/phate.py:826, in PHATE.fit(self, X)
    824 if self.graph is None:
    825     with _logger.task("graph and diffusion operator"):
--> 826         self.graph = graphtools.Graph(
    827             X,
    828             n_pca=n_pca,
    829             n_landmark=n_landmark,
    830             distance=self.knn_dist,
    831             precomputed=precomputed,
    832             knn=self.knn,
    833             knn_max=self.knn_max,
    834             decay=self.decay,
    835             thresh=1e-4,
    836             n_jobs=self.n_jobs,
    837             verbose=self.verbose,
    838             random_state=self.random_state,
    839             **(self.kwargs)
    840         )
    842 # landmark op doesn't build unless forced
    843 self.diff_op

File ~/.local/lib/python3.8/site-packages/graphtools/api.py:259, in Graph(data, n_pca, rank_threshold, knn, decay, bandwidth, bandwidth_scale, knn_max, anisotropy, distance, thresh, kernel_symm, theta, precomputed, beta, sample_idx, adaptive_k, n_landmark, n_svd, n_jobs, verbose, random_state, graphtype, use_pygsp, initialize, **kwargs)
    256     else:
    257         msg = msg + " and PyGSP inheritance"
--> 259 _logger.log_debug(msg)
    261 class_names = [p.__name__.replace("Graph", "") for p in parent_classes]
    262 try:

AttributeError: 'TaskLogger' object has no attribute 'log_debug'
```

`phate.__version__`
'1.0.10'

Can you please let me know what is the reason for this error?
Thanks a lot

**Comment by scottgigante** [comment]
What version of `tasklogger` are you running? I have updated PHATE to require >=1.2.0.

**Comment by danli349** [comment]
@scottgigante I fixed the error by upgrading `tasklogger` to 1.2.0. Thanks a lot!
